clients/web/src/templates/widgets/jsonMetadatumEditWidget.jade

```diff
@@ -0,0 +1,13 @@
+input.input-sm.form-control.g-widget-metadata-key-input(type="text", value=key, placeholder="Key")
+
+#json-editor
```

Comment:
Let's use a class here instead of an ID, since ID's are supposed to be globally unique and this is a widget that we may want to instantiate multiple times. Also let's prefix our classes with `g-` to avoid naming collisions.